sci_strategic_plugin/src/sci_strategic_plugin.cpp

```diff
@@ -347,10 +347,10 @@ bool SCIStrategicPlugin::planManeuverCb(cav_srvs::PlanManeuversRequest& req, cav
       double desired_deceleration = config_.vehicle_decel_limit * config_.vehicle_decel_limit_multiplier;
 
 
-      double safe_distance = pow(current_state.speed, 2)/(2*desired_deceleration) + config_.min_gap;
+      double safe_distance = pow(current_state.speed, 2)/(2*desired_deceleration);
```

Comment:
Is the min_gap parameter no longer used? Should it be removed from the node?

Reply:
it is still used by the status and intent msg sent to carma streets. It was also used as extra space for the safe distance, but that is not needed anymore.